src/user/components/LoginForm/LoginForm.tsx

```diff
@@ -1,4 +1,4 @@
-import { ChangeEvent, useState } from 'react';
+import React, { ChangeEvent, useState } from 'react';
```

Comment:
We dont need to import the full React object, we can use granular named import like we do with ChangeEvent

Reply:
React object removed.